routes/flock.json

```diff
@@ -119,43 +119,282 @@
+        ],
+        "parallax": "/images/flock/goose4/frame.png"
+      },
+      {
+        "title": "Shell",
+        "content": [
+          "Designed with the focus on aesthetics, mass and aerodynamics, and inspired by the F-117 bomber, our smoked polycarbonate shell covers the entire pod while still allowing for its individual components to be shown and easily tweaked. Our GOOSE IV shell was also designed to stand out, with ample space to display our logo as well as promotional material."
+        ],
+        "parallax": "/images/flock/goose4/g4-shell.png"
+      },
+      {
+        "title": "Guidance",
+        "content": [
+          "Guiding the pod along the track, the guidance system provides lateral stability (so the pod doesn't tilt to one side) and controls vertical suspension. This system is split into the front and back of the pod, with the front holding the pod up and absorbing shocks from the track, and the back preventing the pod from derailing."
+        ],
+        "parallax": "/images/flock/goose4/guidance.png"
+      },
+      {
+        "title": "Power",
+        "content": [
+          "The power system comprises of 2 parts: High Voltage system (HV), and Low Voltage system (LV). The HV system powers the linear induction motor, while the LV system powers all other components on the pod. An enclosure is used to insulate the batteries used in the 2 systems, and a Battery Management system ensures that consistent and stable power is provided to the pod."
+        ],
+        "parallax": "/images/flock/goose4/power.png"
+      },
+      {
+        "title": "Motor Control and Inverter",
         "content": [
-          "This is GOOSE III pod, the one that we are building in the next few terms to get ready for Competition III."
+          "The primary way of controlling the operating speed of the Linear Induction Motor (LIM) is to use an inverter to adjust the AC output frequency to the LIM using the DC battery power supply. The controller is used to ensure a stable power output from the non-constant three-phase AC loead from the LIM, while also accounting for the end effects (or \"turbulence\") produced by the LIM."
+        ],
+        "parallax": "/images/flock/goose4/motor-and-inverter.png"
+      },
+      {
+        "title": "Software and Embedded",
+        "content": [
+          "Designed with modularity and minimalism as a focus, the Software and Embedded team incorporates redundant and inexpensive devices that work together to control the pod. Multiple Raspberry Pis are used to aggregate data, communicate with the Arduinos through the CAN bus system and perform other complex tasks."
         ]
       }
     ]
   },
   {
     "name": "Goose V",
-    "thumbnail": "/images/flock/goosex.png",
+    "thumbnail": "/images/flock/goose5.png",
     "url": "goose_v",
+    "desc": "UnLIMited potential.",
```

Comment:
Is this on purpose? Because of the LIM stuff from goose 4?

If not, can you put it back to Unlimited. TBH, i would just leave it as Unlimited, because I feel like many people will. just click this tab without reading goose 4